light/data_model/db/episodes.py

```diff
@@ -29,14 +39,17 @@
 if TYPE_CHECKING:
     from light.graph.structured_graph import OOGraph
 
-SQLBase = declarative_base()
 FILE_PATH_KEY = "episodes"
 ID_STRING_LENGTH = 40
 USR_KEY = DBPlayer.ID_PREFIX
 MAX_WILD_MODEL_LEN = 200
 MAX_WILD_CHOICE_LEN = 100
 
 
+class SQLBase(DeclarativeBase):
```

Comment:
Just a question, why not using `DeclarativeBase` directly? Do you plan to extend it later, or just having a more descriptive name?

Reply:
You're not allowed to extend `DeclarativeBase` directly due to some SQLAlchemy metaclass magic